client/src/components/Footer.js

```diff
@@ -1,34 +1,50 @@
 import React from "react";
 import "../styles/Footer.css";
 import CYFLogo from "../assets/CYF.jpeg";
-import { Divider, Grid, Image, Segment } from "semantic-ui-react";
+import {
+  Container,
+  Grid,
+  Image,
+  Segment,
+  List,
+  Header,
+  Divider
+} from "semantic-ui-react";
 
 const Footer = () => {
   return (
-    <div className="footer">
-      <Grid divided columns={3}>
-        <Grid.Column>
-          <div className="team-container">
-            <h3>Min-Imece-Team</h3>
-            <span>Samuel</span>
-            <span>Hacer</span>
-            <span>Fatma</span>
-            <span>Mawaddah</span>
-            <span>Natalia</span>
-          </div>
-        </Grid.Column>
-        <Grid.Column>
-          <p>
-            A dashboard for Maryhill Integration Network that staff can use to
-            better track, and support service users and volunteers created by
-            CodeYourFuture Students
-          </p>
-        </Grid.Column>
-        <Grid.Column>
-          <img class="footer-image" src={CYFLogo} alt="CYFLogo" />
-        </Grid.Column>
-      </Grid>
-    </div>
+    <Segment className="footer" inverted vertical textAlign="center">
+      <Container textAlign="center">
+        <Grid divided columns={3}>
+          <Grid.Column width={4}>
+            <Header inverted as="h4" content="Min-Imece-Team" />
+            <List link inverted>
+              <List.Item as="a">Samuel</List.Item>
```

Comment:
Do we want to link your names to a web page? If not, let's remove the `as="a"` since we're not linking to anywhere